Azure/sonic-buildimage
# [202111] Add sonic_release file in order to have "release 202111" on sonic_version.yml

#### Why I did it

Fix [Issue 9972: Incorrect information about release version in sonic_version.yml](https://github.com/Azure/sonic-buildimage/issues/9972)

#### How I did it

Add "sonic_release" file to /sonic-buildimage/files/image_config/

#### How to verify it

Install the image and run: `cat /etc/sonic/sonic_version.yml`
verify the following item on sonic_version.yml file:
`release: '202111'`

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
